com.unity.formats.fbx/Documentation~/export-record-in-play-mode-recorder-clip.md

```diff
@@ -0,0 +1,32 @@
+# Record FBX animations via a Timeline Recorder Track
+
+Set up one or multiple Recorder clips in a Timeline Recorder Track to activate the recording of GameObject's animations to FBX at specific time or frame intervals in Play mode.
+
+>**Note:** This scenario requires to have both the [Recorder](https://docs.unity3d.com/Packages/com.unity.recorder@latest) and [Timeline](https://docs.unity3d.com/Packages/com.unity.timeline@latest) packages installed in addition to the FBX Exporter package.
+
+To record an FBX animation through a Timeline Recorder Track:
```

Comment:
suggestion: it could be nice to have an image here too of the timeline track/settings